testbed/pyproject.toml

```diff
@@ -37,6 +37,11 @@ requires = [
     "../core",
     "../travertino",
 ]
+# Look for pre-release wheels in the dist folder of the parent repo.
+requirement_installer_args=[
+    "--pre",
```

Comment:
The `--pre` option is needed so that the Textual builds find the pre-release Travertino; but that also means the pygobject 3.51.0.dev0 pre-release is found. So - toga-gtk also needs to pin `pyobject==3.50.0`. The long term fix for #3143 will likely involve relaxing this pin.

Reply:
There should be a comment explaining which wheels we're talking about here, when they would be used, and why.